src/main/java/frc/robot/subsystems/BallPath/Shooter/ShooterImpl.java

```diff
@@ -38,11 +108,253 @@ public int checkBalls(){
     }
 
     @Override
-    public void setHoodAngle(double distance){}
+    public void setHoodAngle(double setPointHood){
+        if(turretHoodPosition >= setPointHood - 20000 && turretHoodPosition <= setPointHood + 20000){
+          hoodMotor.set(ControlMode.PercentOutput, 0);
+          hoodReady = false;
```

Comment:
Should this be `hoodReady = true` since in this clause we are setting the motor speed to 0 because we are within tolerance of the setpoint position?

Reply:
Yeah, my mistake

Reply:
This is still being set to false